packages/react/src/PageHeader/PageHeader.tsx

```diff
@@ -337,6 +337,7 @@ const TitleArea = React.forwardRef<HTMLDivElement, React.PropsWithChildren<Title
     )
   },
 ) as PolymorphicForwardRefComponent<'div', TitleAreaProps>
+TitleArea.displayName = 'TitleArea'
```

Comment:
This is great, thanks for adding this! We initially had all sub components FC so I think React was able to infer their names but then we refactored a couple if them. 

Do you mind adding the same thing to Parent Link? 🙏🏻

Reply:
Not at all, updated! Do you think we need it for `Root`, too?

Reply:
I think we have a displayName for Root

```
export const PageHeader = Object.assign(Root, {
  ContextArea,
  ParentLink,
  ContextBar,
  TitleArea,
  ContextAreaActions,
  LeadingAction,
  Breadcrumbs,
  LeadingVisual,
  Title,
  TrailingVisual,
  TrailingAction,
  Actions,
  Description,
  Navigation,
})

PageHeader.displayName = 'PageHeader'
```